tmp/build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -0,0 +1,207 @@
LhIojCg4q+0gLA0R481Y3lr7YP2CtUelkJYGlkUvsHz/QBWRddzU6V3eCAszaPOCotJxvSb329V6cNhag9L1OZ/g/O9MZcvFV6rKsH3+1NL7WE9vVuF7+MqZaS3GuvvTydVhyVq87/uwP7tS1cvl3jsZtx/Cm1UHdbQnUpzTo6anqpvHV69YZkMItarq8+1N7rNKnt3iHrD6m/24r66+kQShZxFTAmVhmUyiMNdOf3raaDFZ9Kr6ENLpWHdu1mWs4+rFItJblXTi5bVGJbJID7uJnA1/eRuVb5qbvGoMazBFl8hiS9acpaYOprUsg6vurD0sjYHZ1cnUc8TraoL62F7OZGPq5Srl0tqvIWk/vRcUldYeln7/6RvsDzpriI7QG6oK6z7bT6dRPgR+KYv7KVFBrIz5ICKwtLL2qc95O8HH61/8X9YqqKw+pu9lBxR9rd4jXRMgo9awpJFYaSPlmf9RK1w79Yl0lPgpZawHnVWUPJxlfL0bnU+12elqghLpu8ecIFZ7uX6TEsVYWHdakc2yPU6PP9h4d5qR1bhLmiqsZGeAhf+w8rBVjuyhx3cfg8P52HlZqsd2Y06R5lDIT0FFpyHlbOtdmS8rsPzHFYut9qRtTW4LHSnj4bnsHK51Y5M1ApDXbTXj4DbsHK/1Y6sv5nDFy1zG1but9qRmY3SAHfvw+UzLFJb7cged1cJfN3i4TMsUlvtyEqLjV99YSc9RTZxGBbZrXZknF3CcxgW2a12ZPWVJXXl/Hz/L29h0bDVjmy4m58PLd7ComGrHVlTjd1RbCQ9RXZwFRYlW+3IJFEYov5WwTlxFRY9W+3I2htdJi5eeMRPWFRttSNTjHIvF1+lyU9YtG21I3vaU53P/mEpJ2FRuNWOTDFKnY1u0lNcFCdh0bnVjuwx+0taPIRF7VY7Mp/TwvodHh7ConmrHdlDBu9KfY75sCjfakd2vdbuczG8Ds98WPRvtSMbaGH4wUO2w2Jiqx1Z7y1vgZnVv+LZDouJrXZkojZ/mNldGobDYmirHVnfbS+jL1pmOCyGttqRmQwSo694YDUs5rbakQ2z+a/JaljMbbUjs1sN12vZe3cIk2ExutWOjMWdWCbDYnSrHVldefGVCsbW4dkLi+mtdmSDrYy98Ii9sJjeakfW8qXDZTORniIDjIXF+lY7MlHL2Do8Y2FxsNWOrK3BxdD3/7IUFh9b7cjMBqmfnXeHsBQWN1vtyPpbfILAxh0eZsLiaasdWWmR8U6Th/QU58JMWJxttSNj5UYWG2Hxt9WOrMyh1LNwWMpGWFxutSNj4icYBsLidasdWUO1rcJtIT3FGRgIi+OtdmSD1K/D0x4W31vtyHpulRUqVF8b0B4W31vtyLRC/n26b0tTfTcX31Z7MqmZD36Yf7++FNmeXoiuxHZ//c/84Y9/NepFq6LzORWb1SBLQlmp4ik1y6KAY6RMPWgv/9NfXiWSH0kPkh7VYWVrqz2RTM6+XXsTWg+ubL98E42s7a5+SFPSMeHIdup/PJ9e+vz/z9NobFaD2ShZTJLXYXEUGyRR8NjNZQ5FL+fuv6fZIN1vK//z3+Zz9idmhN6wkLfa44nkq8XY26XN0Op2bGN3cjYSWd/L4mAfNZrl2M5ybEej0fw0s3LsV4ssOp/TUlyo10uCo9gkS4KzxOh1KCZD9n/+GO6qhLAyds6t9sN44uVCLLi69X55M7qxNzkb+bC5n5MB04uu70VP6LjALHsdir3IIIuCy2aStIKj2FjmVApQP5VLCg03r5T+OLV0sZGxoDSsk7ba9w8S75Y3gytbM4uxyNru5FxkY/uAxIAo1jb3/ZuraX/JbBDLnIqz2JRqzutUDLLWatEVWfSn/57/11MNYWUgtdW+s3cYXNl6t/zfjKJruxNzke3dQ9KjYbG5czg1F52ai/76l/Sy1utU3DaTJArOo084xSApJtluNWg0mlqftdxlWQiuk5j6NHl9331PeoY06iuL596t7e7HSQ9CNUkUykrNh/HkYniD9CzHUfqJNTkXIT0CAw4OE7Pv1khPkR7tB6SAURAWwALCAlhAWAALSi/ea71Wr5PhN3DmzP5hYiwQ2j9IkB7kOErDmlmM2az6x91VlZ4C0rPQKBTZGvOHpuejP07TeDpK7znWJ+0Nrt8O1LrtZtKDkJdMambffRgLBMcCodDqNulxzkB7WCnNVx2/e1DnUWVeW7uHE7Or44HQWCC0R99feSeh9K/CY8YnwuMT4Rt1pb9/8IXXSfu6d1aEI9tjgeCLlyuB2fS3FynHRlgpz6eXXswsN191fNtXw2Ve8XhyZjE2FghNzUfm31N3+y8jLIV1dJ3xcdQfGp8Idza6h7oqK9w8XNqvbe5Pz0eeTy/99GrlpJUb5rBxjZVWfn5e6zXnE2Z/cny7tBF4vToWCE8vROOJJOlxsozhsD5pa3B921dT5mDj3GvmTez59NKoP/h+ZYv0LBjxEFZKW4NrpLfaR+W112E8MeoP/TAZnni9us7OZuJF8BNWStdXnsfdlynJazm2PfrP0PRC9NlU+COlT9PgwltYKR2NLlKn9h81Gv+rFf/s6rOJ8NvlzdwPQAk+w0rJ5an9zt7h5Fxk7OgYc2cPFl+5DisF66n9cmxnPBB6MbP8j1fHnwNTOcbOsRBk/dQ+nkj+698fxgKhUX/wPA++qhP/YaVc/NR+fWvf/3p1PBCaWYxBT2dSS1jIp/bvlzcn5iKj/uD0fPQgztsxJj78X2Oldfqp/f5hYmYhOjUf/bs/+HZJvT/ZXYRKw/rk81P7eDw5NhF6Nhn2v15dI/qcPgfUHlbK7XrHxvbBzJsorS8FYo+KrrFO8cNkmPQIvIGndAAWEBbAAsICWEBYAAsIC2ABYQEsICyABYQFsICwABYQFsACwgJYQFgACwgLYAFhASwgLIAFhAWwgLAAFhAWwALCAlhAWAALCAtgAWEBLCAsgMV/AgAA//956aWpNSdLJgAAAABJRU5ErkJggg== 
```

Comment:
will we have different images for community/product?

Reply:
Don't know at this point. It's possible various descriptions in here might end up being different between the two versions too, at which point a template per version (community vs. supported) would be needed.